tests/sweep_framework/sweeps/conv2d/short/conv2d_short_sweep.py

```diff
@@ -1583,6 +1583,372 @@
         ],
         "is_conv1d": [True],
     },
+    "tt-forge_sweep_conv2d": {
+        "input_specs": [
+            # Contains following params
+            # [batch_size, output_channels, input_channels, input_height, input_width, kernel_height, kernel_width, stride_x, stride_y, pad_x, pad_y, groups, bias, dilation, datatype]
+            [1, 1024, 1024, 14, 14, 3, 3, 1, 1, 1, 1, 1, False, 1, int(ttnn.bfloat16)],
```

Comment:
we seem to be missing info about whether tensors are tilted or not

Reply:
tensor **tilted**?? I did not see such info. in sheet. I mainly have derived above from attributes only. Let me know if I am missing something.

Reply:
"input_layouts": [
      {
        "mapping_from": "(d0, d1, d2, d3)",
        "mapping_to": "(d0 * 196 + d1 * 14 + d2, d3)",
        "memory_config": [
          7,
          32,
          "tile<32x32, bf16>",
          "dram"
        ]
      },
      {
        "mapping_from": "(d0, d1, d2, d3)",
        "mapping_to": "(d0 * 3072 + d1 * 3 + d2, d3)",
        "memory_config": [
          3145728,
          3,
          "bf16",
          "system_memory"
        ]
      },
      {
        "mapping_from": "(d0, d1, d2, d3)",
        "mapping_to": "(d0 * 196 + d1 * 14 + d2, d3)",
        "memory_config": [
          7,
          32,
          "tile<32x32, bf16>",
          "dram"
        ]
      }

      In data you have this type of info from this you can infer if tensor is tilized or not if it's on device or on host

Reply:
ahhh....it is **tiled.** 
Since every test case was tiled in json, I thought it is ok to ignore. However, I just checked our tensors are row major, by default. 
will correct it and also add one more field of memory.

Reply:
Not all inputs are flagged as tiled, please do not ignore as this is important. In general, look at the memory config section, it will be explicitly marked as tiled.

Reply:
Note that there is a readme for the JSON file in development.
https://github.com/tenstorrent/tt-torch/pull/139
Add comments if you have questions about any of the fields. But it's critical that we get the tensor format and op arguments in our tests to be exactly what Forge is using.

Reply:
Several point to note here.
-> Weight tensors are missing layout info. I have confired with @AleksKnezevic  and @LPanosTT and they are not giving any specific layout while creating weight tensor. same is for our case.
-> @AleksKnezevic confirmed that these test cases do not have any bias tensors.
-> input and output tensors are of tiled layout. modified code to do same.

Reply:
Now we seem to be missing info if tensor is on host or on device?

Reply:
"system_memory" means host tensor for them as far as I understand

Reply:
-> As of now, in forge input and output buffers are in DRAM. I have added buffer type as DRAM or L1 for them.
-> For weight tensor, They are using default setting which is being used now for us also.(weight buffer's buffer type is ignored.). If I understand correctly, if we do not specify any memory_config, it should be of host only. Do let me know if I am missing something.

I also have asked @AleksKnezevic  and @LPanosTT  for steps(, how they are running) to run test case. Once I have steps, I shall try to run one test case with debug logs enabled and see If I am missing anything.

Reply:
yeah, for now it seems that all the weights are on host, but that will probably change.  It's fine to keep as is for now